### ISSUE TITLE
[REQUEST] Possibility to force system font ?

### ISSUE THREAD

**Issue by xerta555** [ikws4/WeiJu2#9]
Hi :)

Is it possible to force any app to use the system font instead of a specific one ?

Bcz some apps like Twitter nor Github for exemple use their own font and don't prupose the option to let the user decide which font to use.

Thanks in advance.

**Comment by ikws4** [comment]
All the text is rendered by TextView (unless it's a flutter app), you could take a look at it.

**Comment by xerta555** [comment]
> All the text is rendered by TextView (unless it's a flutter app), you could take a look at it.

Even the style font ok

And about `adaptive_color_bar` script, it won't works on my side for few apps like Textra, Telegram X, Magisk, Vivaldi, etc...
I precise that i only have checked `System Framework` in LSPosed app for WeiJu2, maybe that i must enable all apps for which i want to do something with your module ?

**Comment by ikws4** [comment]
Yes, WeiJu2 only works when you enable them in LSPosed. Because the `adaptive_color_bar` script uses the android sdk to change the color, not directly hook the `System Framework` (we cannot do hook `System Framework`, because it start before WeiJu2 was activated).

**Comment by VD171** [comment]
Did you try to change fonts using magisk instead xposed?

**Comment by xerta555** [comment]
> Did you try to change fonts using magisk instead xposed?

Yes, but the custom font injected by Magisk isn't applied in an application which have it own font(s) file(s).
